build/scripts/entrypoint-volume.sh

```diff
@@ -23,7 +23,8 @@ fi
 ls -la /checode/
 
 # Start the machine-exec component in background
-nohup /checode/bin/machine-exec --url '0.0.0.0:3333' &
+export MACHINE_EXEC_PORT=3333
```

Comment:
This was done so that the VS Code extension could determine the machine exec port